src/openstudio_lib/ScriptFolderListView.cpp

```diff
@@ -88,7 +88,7 @@ void ScriptFolderListView::addScriptToFolder(const openstudio::path& t_path, con
   openstudio::filesystem::create_directories(folder);
   openstudio::path filename = folder / t_path.filename();
   filename = iterateFileName(filename);
-  openstudio::filesystem::copy_file(t_path, filename, openstudio::filesystem::copy_option::overwrite_if_exists);
```

Comment:
I guess OS updated to std::filesystem instead of boost::filesystem?

Reply:
no. But boost was updated to 1.79.